e2e_test/streaming/nexmark/q101.slt.part

```diff
@@ -0,0 +1,43 @@
+query ITI rowsort
+select * from nexmark_q101;
+----
+1000 mksxmeugsicmeb 73134520
```

Comment:
Just to confirm, are these results generated by Postgres instead of RisingWave?

Reply:
No, it's actually RW Batch's result.

As `auction_id` is created as the primary key and is always used as ether join key or group key,
I just compared the total number of rows manually against PG's result.

Reply:
Okay. Let's try to be confident. 😄